open-media-match/src/OpenMediaMatch/app.py

```diff
@@ -72,6 +81,36 @@ def status():
         """
         return "I-AM-ALIVE\n"
 
+    @app.route("/", methods=["POST"])
+    def upload():
```

Comment:
blocking: I think we should move the ui to its own blueprint, since I think we'll need one more request to handle the bank add later.